examples/PurePursuitController/purePursuitControllerExample.cc

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
Would you mind changing the names of this folder and the ``*.h`` and ``*.cc`` files to use snake case please? i.e.
``examples/pure_pursuit_controller/pure_pursuite_controller_example.cc``

It's just that that's what we use elsewhere and it's good to be consistent.

Reply:
of course no problem